improver/cli/__init__.py

```diff
@@ -218,6 +218,8 @@ def create_constrained_inputcubelist_converter(*constraints):
             will be sorted to match their order.
             A constraint can be an iris.Constraint object or a callable
             or cube name that can be used to construct one.
+        strict (boolean):
+            If strict is True, then there must be exactly one cube per constraint.
```

Comment:
When would there not be exactly one cube per constraint in the coefficients case?

Reply:
I've removed these changes, as these are no longer necessary following restructuring the beta coefficient cube.